src/styles/main.scss

```diff
@@ -297,6 +297,12 @@ ol {
 }
 
 
+svg {
```

Comment:
Couple of concerns here:

First, I'm not sure I understand why this is necessary. If the rows have a fixed height, then we just wait for subviews to render, and the topmost view checks its total rendered height in the DOM. There's no need to involve images.

Second, it seems funny to globally change the positioning behavior of any element. But here it's extra odd, because the svg tags are "inside" an image file. This makes my brain hurt! I don't think we need to do this anyway (per my first comment), but if we wanted to make this kind of positioning change to some elements, we should do it with a class name--otherwise life becomes rather unpredictable.